wkcuber/api/Layer.py

```diff
@@ -258,6 +267,8 @@ def downsample(
             prev_mag = target_mag
             target_mag = get_next_mag(target_mag, scale)
 
+        self.set_bounding_box(original_bounding_box[0], original_bounding_box[1])
```

Comment:
You also have to fix it for `downsample_mag` and `downsample_mag_list`. (Everywhere where `_pad_existing_mags_for_downsampling` is used)

I have given this another thought and I think it is important to make sure that in case of an error, the bounding box does not get overwritten. 
I have come up with three possibilities:

1. One way would be to avoid exporting the properties to disk for this function call. However, this does not work because the properties are exported automatically occasionally (e.g. when a new mag is added (which is the case for downsampling)).
2. Not updating the properties in the first place (just the views themselves). If I am not mistaken, the only time during downsampling when the bounding box from the properties are used, is when `mag.get_view()` is called (in `downsample_mag`). The best approach would be to remove `_pad_existing_mags_for_downsampling` completely and also the parameter `pad_data` in `downsample_mag_list`. Then in `downsample_mag` you would have to pass the correct offset and size to `get_view()`.
3. Or you wrap the code in a `try` block and then reset the bb in the `except` before you propagate the error anyways. However, this still causes problems if the process is killed completely.

I would encourage you to go for Idea 2. I hope it works the way I imagine it (if you need help, just call me). This would also make the code much nicer because this "padding" might be confusing if someone encounters it for the first time.